modules/social_features/social_user_export/src/Plugin/Action/ExportUser.php

```diff
@@ -87,13 +87,13 @@ public static function create(ContainerInterface $container, array $configuratio
    */
   public function executeMultiple(array $entities) {
     if ($this->applyAll) {
-      $this->tempStoreFactory->get('user_operations_export')->set($this->currentUser->id(), [
+      $this->tempStoreFactory->get('user_operations_export')->set($this->current_user->id(), [
```

Comment:
Hey Brhane , just a reminder that this one is not the correct name. You said you corrected it, but I guess you need to push those changes then.